### ISSUE TITLE
Importing multiple packages on one line broken when using namespace packages installed in editable mode

### ISSUE THREAD

**Issue by isaac-ped** [microsoft/pyright#9165]
**Describe the bug**
If two different packages install into the same namespace in editable mode, pyright reports "Unknown import symbol" if both packages are imported on the same line, but not if they are imported on separate lines.

**Code or Screenshots**

I have two namespace packages set up as follows:
```
./
├── pkg_a/
│   ├── src/
│   │   └── nspkg/
│   │       └── module_a/
│   │           └── __init__.py
│   └── pyproject.toml
├── pkg_b/
│   ├── src/
│   │   └── nspkg/
│   │       └── module_b/
│   │           └── __init__.py
│   └── pyproject.toml
├── broken.py
└── working.py
```

pkg_a/pyproject.toml contains:
```toml
[project]
name = "pkg_a"
version = "0.0.1"
```

pkg_b/pyproject.toml contains:
```toml
[project]
name = "pkg_b"
version = "0.0.1"
```

`working.py` contains:
```python
from nspkg import module_a
from nspkg import module_b
```

and `broken.py` contains:
```python
from nspkg import module_a, module_b
```

With that setup, the following script should reproduce the error:
```bash
python3 -m venv venv
source venv/bin/activate
pip install pyright --editable ./pkg_a --editable ./pkg_b
set -x
pyright working.py
pyright broken.py
```

Which produces the following output:
```
+ pyright working.py
0 errors, 0 warnings, 0 informations
+ pyright broken.py
~/pyright_repro/broken.py
  ~/pyright_repro/broken.py:1:29 - error: "module_b" is unknown import symbol (reportAttributeAccessIssue)
1 error, 0 warnings, 0 informations
```

**VS Code extension or command-line**
Same issue in both vscode (with pylance) and running pyright locally with the CLI.

```
$ pyright --version
pyright 1.1.383
```

And just in case it matters
```
$ python --version
Python 3.10.3
```

Though the same issue persists with other python versions and at least with pyright back to `1.1.316` which is where I first encountered the issue

**Comment by erictraut** [comment]
Pyright doesn't support importing symbols from different namespace packages in the same import statement. See [this issue](https://github.com/microsoft/pyright/issues/3630) for more details. As I mentioned in that issue, supporting this would require a major rewrite of a very complex subsystem of pyright. This rewrite would also likely have some pretty significant performance implications. For those reasons, it's unlikely to happen any time soon, if ever. If there's sufficient demand from pyright users, it's something I might consider, but so far, the other issue has been upvoted only once.

If you want your code to work with pyright, you'll need to separate these imports onto separate lines.
